README.adoc

```diff
@@ -36,10 +36,18 @@ Explore how to externalize configuration using MicroProfile Config and configure
 // =================================================================================================
 
 == What you'll learn
-The https://openliberty.io/guides/microprofile-config.html[MicroProfile Config guide] teaches you how to externalize configuration from your application. It may not be immediately clear why this is beneficial to you when developing your microservices. You will learn how to externalize your configuration, and provide the configuration values by creating environment variables. By using MicroProfile Config to handle your configuration, it provides useful annotations that you may use to inject configured values into your code. These values can come from environment variables or elsewhere.
+The https://openliberty.io/guides/microprofile-config.html[Configuring microservices guide] teaches you how to externalize configuration from your application. It may not be immediately clear why this is beneficial to you when developing your microservices, but {kube} provides some tools that allow you to conveniently configure your microservices with a setup such as the one described. You will learn how to externalize your configuration, and provide the configuration values by creating environment variables. By using MicroProfile Config to handle your configuration, it provides useful annotations that you may use to inject configured values into your code. These values can come from environment variables or elsewhere.
 
 Furthermore, you'll learn how to set these environment variables using ConfigMaps and Secrets. These resources are provided by {kube} that act as a data source for your environment variables. By creating a ConfigMap or Secret, they can be used to set environment variables for any number of containers as you see fit.
 
+== Prerequesites
```

Comment:
This prereqs content seems similar, if not the same as the kube-intro guide. I'd say we can create a common file to be shared for the kube series of guides?

Reply:
Prerequisites *

Reply:
I added shared docs, I'll update this PR as soon as the other one is merged: https://github.com/OpenLiberty/guides-common/pull/172